### PR TITLE
Enable multiple files upload from upload dialog box

### PR DESCRIPTION
**Pull request by rga-odoo**
task:https://www.odoo.com/web#id=23304&view_type=form&model=project.task&action=327&menu_id=
pad: https://pad.odoo.com/p/r.c33035bc2a298b8c4a4a48682962afc4

enable multiple files upload for chatter, many2many_binary widget and sidebar attachments

### CODE REVIEW

File: addons/web/static/src/xml/base.xml [hunk 1]
```diff
@@ -1033,23 +1035,24 @@
...
                 </div>
                 <a class="o_image" target="_blank" t-att-href="file.url"
-                   t-att-title="(file.name || file.filename) + ((file.date)? (' \n(' + file.date + ')') : '')"
+                   t-att-title="file.name"
                    t-att-data-mimetype="file.mimetype"
                    t-attf-data-src="/web/image/#{file.id}/100x80"/>
             </t>
         </div>
+        <div t-foreach="widget.files_uploading" t-as="file" class="oe_attachment">
+            <div>Uplading...</div>
```
Comment (Yenthe666) [inline addons/web/static/src/xml/base.xml]: @rga-odoo Uploading instead of Uplading.

**Comment by Yenthe666** [comment]
@rga-odoo this is the best PR I've seen in weeks, great addition!
@mart-e couldn't we get this in V10 already? It would be such a neat improvement..

**Comment by mart-e** [comment]
@Yenthe666 sorry, new features should always target the master version. We need to apply the same rules to us than the one we apply to the community ;-)

**Comment by jholze** [comment]
👍  very cool.

**Pull request merged**